scripts/client_test/cmds/eval_cmd.py

```diff
@@ -40,35 +42,24 @@ def run(
         :param resource_pool: [ONLY Cloud] which nodes should job run on
         :return:
         """
-        _args = [CLI, self._cmd, "run", "--model", model]
-        for ds in datasets:
-            _args.extend(["--dataset", ds])
-        if version:
-            _args.extend(["--version", version])
-        if runtime:
-            _args.extend(["--runtime", runtime])
-        if name:
-            _args.extend(["--name", name])
-        if desc:
-            _args.extend(["--desc", desc])
-        if step_spec:
-            _args.extend(["--step-spec", step_spec])
-        if resource_pool:
-            _args.extend(["--resource-pool", resource_pool])
-        if use_docker:
-            _args.append("--use-docker")
-        # TODO: return value is str
-        if gencmd:
-            _args.append("--gencmd")
-        if step:
-            _args.extend(["--step", step, "--task-index", str(task_index)])
-
-        _args.extend(["--project", project])
-        _ret_code, _ = invoke(_args, raise_err=True)
 
-        return bool(_ret_code == 0)
+        return JobTermView.run(
```

Comment:
the command line can test the definitions of the click options.

Reply:
This relys on -o json being implemented in bundle building process.